parking_permits/talpa/order.py

```diff
@@ -46,6 +46,9 @@ def _create_item_data(cls, order, order_item):
             "productName": order_item.product.name,
             "productDescription": cls._get_product_description(order_item.product),
             "unit": "kk",
+            "startDate": timezone.localtime(order_item.permit.start_time).strftime(
```

Comment:
It could be that I missed something, but this does not seem quite right to me 🤔   Fixed period permits can have multiple order items (e.g. different prices in different time periods), in which case the start time will be different for them. I think it's also the same for open-ended permits, we will create new orders (and new order items) for each month, and I think the start time should be different for each new order item?

Reply:
But this is not related to the start time of order_item. It's the start time of the permit they belong to. `order_item.permit.start_time`

Reply:
oh, I see